docs/rsdl/rapid-rsdl-paths.md

```diff
@@ -0,0 +1,103 @@
+---
+id: rsdl-paths
+title: RAPID Paths
+---
+
+# Paths in a RAPID Service
+
+The set of valid paths for a RAPID service is the result of applying common path conventions to the model, as described below. You can optionally specify an subset of these paths that are supported by your service using [explicit paths](rapid-rsdl-intro.md#explicit-paths) in your RSDL.
+
+- [Service Root](#service-root)
+- [Key Segment](#key-segment)
+- [Structured Properties](#structured-properties)
+- [Nested Operations](#nested-operations)
+
+## Service Root
+
+The [service](./rapid-rsdl-intro.md#defining-a-service) element in RSDL defines the segments that are available at the service root:
+
+```rsdl
+service {
+  company: Company
+}
+```
+
+## | supported path |
+
+`/company`
+
+## Key Segment
+
+Path segments that represent a collection of [structured types](rapid-rsdl-intro.md#defining-a-structured-type) with identity may be followed by a segment containing the key value of an instance within the collection:
+
+```rsdl
+service {
+  competitors: [Company]
+}
+
+type Company
+{
+  key stockSymbol: String
+}
+```
+
+## | supported paths |
+
+`/competitors`
+`/competitors/{stockSymbol}`
+
+where `{stockSymbol}` is a string value representing the stock symbol of a competitor.
```

Comment:
It is unclear whether "supported paths" means "something you can use within the curly braces after the `paths` keyword, or whether it means "a supported URL".

If the latter, it is better to replace `{stocksymbol}` with an actual value, for example `cgwsl`.

Reply:
clarified that this is a placeholder for the stock symbol.